openedx/features/enterprise_support/tests/factories.py

```diff
@@ -0,0 +1,78 @@
+"""
+Factoryboy factories.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from uuid import UUID
+
+import factory
+from faker import Factory as FakerFactory
+
+from django.contrib.sites.models import Site
+
+from enterprise.models import (
+    EnterpriseCustomer,
+    EnterpriseCustomerUser,
+)
+
+FAKER = FakerFactory.create()
+
+
+class SiteFactory(factory.django.DjangoModelFactory):
```

Comment:
You don't need this factory anymore.

Reply:
Currently I am referring in `EnterpriseCustomerFactory`

Reply:
Okay, then use the version of this factory that already exists [here](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/site_configuration/tests/factories.py).